client/js/transports/small-webrtc.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: "SmallWebRTCTransport"
+description: "A lightweight WebRTC transport for peer-to-peer connections with Pipecat"
+---
+
+The `SmallWebRTCTransport` class provides a WebRTC transport layer for connecting to Pipecat's `SmallWebRTCTransport` server-side implementation. It handles audio/video device management, WebRTC connections, and real-time communication between clients and bots.
```

Comment:
Is it worth putting here that it's meant to be paired with the pipeline's `SmallWebRTCTransport` and link to its docs? (assuming those exist)